main/libcroco/APKBUILD

```diff
@@ -1,13 +1,12 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libcroco
-pkgver=0.6.12
-pkgrel=1
+pkgver=0.6.13
+pkgrel=0
 pkgdesc="GNOME CSS2 parsing and manipulation toolkit"
 url="http://www.gnome.org"
 arch="all"
-license="LGPL-2.1-only"
-subpackages="$pkgname-dev"
-depends=
+license="LGPL-2.1-or-later"
```

Comment:
Please mention the reason for the license change in your commit message.

Reply:
fixed